### PR TITLE
feat(wc): fds-icon-bar

### PR DESCRIPTION
**Pull request by dorsaffrigui**
fix(#802)

### CODE REVIEW

**Comment by Pi5rr5** [comment]
The tooltip should be on the bottom-right
image
image

**Comment by Pi5rr5** [comment]
We should have the icon bar with and without labels
image

**Comment by dorsaffrigui** [comment]
> We should have the icon bar with and without labels image

it's already the case, when removing the label attribute from the fds-icon-bar-item

File: libs/web-components/icon-bar/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+### Import
+
+```
+npm i @finastra/icon-bar
+```
+
+```ts
+import '@finastra/icon-bar';
+...
+<fds-icon-bar></fds-icon-bar>
```
Comment (bcldvd) [inline libs/web-components/icon-bar/README.md:19]: Could we have a meaningful example here and below ?

Such as this one:
```
<fds-icon-bar-item data-tippy-content="This is a tooltip" current icon="home" notification="2"></fds-icon-bar-item>
            <fds-icon-bar-item label="Account" data-tippy-content="This is a tooltip" icon="credit_card"></fds-icon-bar-item>
            <fds-icon-bar-item label="Calendar" data-tippy-content="This is a tooltip" icon="event" notification="1"></fds-icon-bar-item>
            <fds-icon-bar-item data-tippy-content="This is a tooltip" slot="footer" icon="add"></fds-icon-bar-item>
          </fds-icon-bar>
```

**Comment by dorsaffrigui** [comment]
> > We should have the icon bar with and without labels image
> 
> Resolved

**Comment by dorsaffrigui** [comment]
> The tooltip should be on the bottom-right, also can we apply the same styling as in the design ? image image

Resolved

**Review by bcldvd — APPROVED** [review]

**Pull request merged**